charts/vector/values.yaml

```diff
@@ -206,6 +206,10 @@ tolerations: []
 # rules for Vector Pods.
 affinity: {}
 
+# topologySpreadConstraints -- Configure [topology spread constraints](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/)
+# for Vector Pods.
```

Comment:
```suggestion
# for Vector Pods. Valid for the "Aggregator" and "Stateless-Aggregator" roles.
```

Reply:
Added in https://github.com/vectordotdev/helm-charts/pull/261/commits/505056d25a8d19a2956df9ccd34d627197a8a45d and regen'd the docs.